src/layer/vector/CircleMarker.js

```diff
@@ -21,7 +21,12 @@ export var CircleMarker = Path.extend({
 
 		// @option radius: Number = 10
 		// Radius of the circle marker, in pixels
-		radius: 10
+		radius: 10,
+
+		// @option title: String= 'CircleMarker'
+		// Alternative text for a Figure. Used as `<title>` tag in SVG or `title` attribute in Canvas renderer.
+		// Useful for accessibility.
+		title: 'CircleMarker',
```

Comment:
**nit**: What about making this default to `'Circle marker'` instead? It seems a more appropriate thing to expose to a human interface. Since most of the default `title` and `alt` values in Leaflet are a single word, there aren't many benchmarks that I could point to, except maybe the default titles for [zoom in](https://leafletjs.com/reference.html#control-zoom-zoomintitle) and [zoom out](https://leafletjs.com/reference.html#control-zoom-zoomouttitle) buttons.
```suggestion
		// @option title: String= 'Circle marker'
		// Alternative text for a Figure. Used as `<title>` tag in SVG or `title` attribute in Canvas renderer.
		// Useful for accessibility.
		title: 'Circle marker',
```

Reply:
:+1: - Plus one to `'Circle marker'` as the default. Spaces and plain English words are good for assistive technology!

Thank you all for your contributions to this ❤️ 

(And, take my comment above with a pinch of salt - I'm late to the party!)

Nick